src/status_im/contexts/wallet/effects.cljs

```diff
@@ -8,7 +8,9 @@
  :effects.wallet/create-account-from-mnemonic
  (fn [{:keys [secret-phrase keypair-name]}]
    (native-module/create-account-from-mnemonic
-    {:MnemonicPhrase (string/join " " secret-phrase)}
+    {:MnemonicPhrase (if (string? secret-phrase)
```

Comment:
should we make this safer? 🤔 
i.e if it's a string do this, if it's a vector use string/join and otherwise return an empty string/nil? 🤔

Reply:
But we shouldn't get to that point with an empty string. It will be of no use and the status-go call will fail. So probably better to make it fail earlier?

Reply:
tbh, I'm not sure. Just flaggin this to be careful. Whatever you think is best! 🙏